src/layout/internal/TabbingLayoutContainer.ts

```diff
@@ -44,6 +44,30 @@ export default class TabbingLayoutContainer extends AParentLayoutContainer<ITabb
       }
       return true;
     }, null, true);
+
+    if(this.options.fixed) {
+      this.header.classList.add('frozen');
+
+      const mouseEnter = () => {
+        this.header.classList.remove('hidden-header'); // show full header when hovering over the minimal header
+      };
+
+      const mouseLeave = () => {
+        this.header.classList.add('hidden-header'); // hide header again
+      };
+
+      this.on(LayoutContainerEvents.EVENT_LAYOUT_CHANGED, () => {
+        if (this.children.length < 2) { // frozen layout to apply minimal style to the header and hide views
```

Comment:
please evaluate the current state also during intialization, e.g. the constructor already allows to set some children.